components/NotificationPopup/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+
+import Image from "next/image";
+
+import classNames from "classnames/bind";
+
+import { useGetAllNotification } from "@/hooks/useGetAllNotification";
+
+import styles from "./NotificationPopup.module.scss";
+import BaseNotification from "../Notification/BaseNotification";
+import EmptyNotification from "../Notification/EmptyNotification";
+
+const cn = classNames.bind(styles);
+
+export default function NotificationPopup() {
+  const [isClicked, setIsClicked] = useState(false);
+
+  const { data: notificationInfo } = useGetAllNotification();
+
+  const getNotificationCount = () => {
+    if (!notificationInfo) return;
```

Comment:
```suggestion
    if (!notificationInfo){
     return;
    }
```
P4: 저희 컨벤션에 맞게 1줄이여도 중괄호로 사용하는게 좋을 것 같아요!